Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -104,7 +104,7 @@ internal List<Result> GetDefaultHotKeys()
 
         internal async Task InstallOrUpdate(UserPlugin plugin)
         {
-            if (PluginExists(plugin.ID))
+            if (PluginExists(plugin.ID) || Directory.Exists(Path.Combine(DataLocation.PluginsDirectory, plugin.Name)))
```

Comment:
Blocks user from installing the same plugin manually without uninstalling first.